docs/2.9.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -605,46 +683,78 @@ the upgrade test to understand the necessary downtime required for the migration
 Furthermore, upon reconnecting, the participant needs to recompute the new set of commitments. This can take
 a while for large numbers of contracts.
 
-Soft Synchronization Domain Connection Upgrade
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _one_step_migration:
 
-.. note ::
+One-Step Migration
+------------------
+A *one-step migration* intends to :ref:`upgrade the binary <upgrade_canton_binary>` and
+:ref:`change the protocol version <change_pv>` in one step. It is a combination of the
+steps documented in :ref:`upgrade_canton_binary` and the :ref:`hard_sync_domain_upgrade`.
+Thus, strong understanding of these sections is required.
```

Comment:
I'd remove that section: for me the user should be able to follow the steps **carefully** without necessarily understanding everything which is behind.